mysql/errname.go

```diff
@@ -916,6 +916,16 @@ var MySQLErrName = map[uint16]string{
 	ErrForUpdateCantRetry: "[%d] can not retry select for update statement",
 	ErrAdminCheckTable:    "TiDB admin check table failed.",
 
+	ErrInvalidPluginID:            "Wrong plugin id: %s, valid plugin id is [name]-[version], both name and version should not contain '-'",
+	ErrInvalidPluginManifest:      "Can not read plugin %s's manifest",
+	ErrInvalidPluginName:          "Plugin load with %s but got wrong name %s",
+	ErrInvalidPluginVersion:       "Plugin load with %s but got %s",
+	ErrDuplicatePlugin:            "Plugin [%s] be duplicate declared in init parameters",
```

Comment:
```suggestion
	ErrDuplicatePlugin:            "Plugin [%s] is redeclared",
```

I'm not sure how "declared in init parameters" is relevant besides this error is emitted in the `plugin.Init()` function, so I've removed the last clause in the suggestion.